elasticgraph-indexer_autoscaler_lambda/lib/elastic_graph/indexer_autoscaler_lambda/concurrency_scaler.rb

```diff
@@ -37,6 +38,8 @@ def tune_indexer_concurrency(queue_urls:, min_cpu_target:, max_cpu_target:, maxi
 
         new_target_concurrency =
           if num_messages.positive?
+            free_storage = get_min_free_storage(cluster_name)
```

Comment:
It'd be good to use units consistently on these. Assuming they are all in megabytes:

* `minimum_free_storage` -> `min_free_storage_in_mb` (I think `min` instead of `minimum` is just as readable).
* `free_storage` -> `free_storage_in_mb`
* `get_min_free_storage` -> `get_min_free_storage_in_mb`

Also, I'm realizing that `min` is being used in two distinct ways in this code:

* It refers to the configured threshold in one usage
* In the other usage it is qualifying the free storage space value we get back from cloudwatch to indicate it's the value from the node with the least space.

It'd be good to not overload `min`.

WDYT about using alternate terms for these?

* `get_lowest_node_free_storage_in_mb`
* `required_free_storage_in_mb`

I particularly like using `required` for the configured threshold.

Thoughts?

Reply:
Yeah it's definitely overloaded (your comment about not being sure which one was logged highlighted that). Will put the unit in as well to make that more clear